src/components/home/DdayBadge.module.scss

```
@@ -10,6 +11,30 @@
+
+  // 접수 기간에만 나타나는 배지라 등장 자체가 정보다 — 아래에서 통통 떠오르며 시선을 끈다.
+  animation: dday-pop 500ms $ease-out backwards;
+  animation-delay: 400ms;
+
+  @media (prefers-reduced-motion: reduce) {
+    animation: none;
+  }
+}
+
+@keyframes dday-pop {
+  from {
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)